src/components/Notification/NotificationWithIcon.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { Toast, ToastHeader, ToastBody, Button } from 'reactstrap'
+import Icon from '../Icon/Icon'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+const propTypes = {
+  header: PropTypes.string,
+  content: PropTypes.string,
+  state: PropTypes.string,
+  dismissable: PropTypes.bool,
+  fix: PropTypes.string
+}
+
+// custom style
+const NotificationStyle = {
+  right: 'auto',
+  bottom: 'auto',
+  top: 'auto',
+  left: 'auto'
+}
+
+const NotificationWithIcon = props => {
```

Comment:
I would unify `NotificationWithIcon` and `NotificationStandard` under the same component `Notification`.

Reply:
done 👍